src/components/Wallet/ConfirmWallet.jsx

```diff
@@ -10,57 +10,115 @@ import {
   Box,
 } from "@material-ui/core";
 import { Alert } from "@material-ui/lab";
+import { MultisigWalletPolicy } from "unchained-wallets";
+import { getWalletConfig } from "../../selectors/wallet";
 
-class WalletConfirmation extends React.Component {
-  render = () => {
-    const { startingAddressIndex } = this.props;
+const ConfirmationInfo = () => {
+  const extendedPublicKeyImporters = useSelector(
+    (state) => state.quorum.extendedPublicKeyImporters
+  );
+  return (
+    <>
+      {Object.values(extendedPublicKeyImporters).map((importer) => (
+        <TableRow key={importer.extendedPublicKey}>
+          <TableCell>{importer.name}</TableCell>
+          <TableCell>
+            {importer.method === "text" ? "N/A" : importer.bip32Path}
+          </TableCell>
+          <TableCell>{importer.extendedPublicKey}</TableCell>
+        </TableRow>
+      ))}
+    </>
+  );
+};
+
+const PolicyInfo = ({ keys }) => {
+  const extendedPublicKeyImporters = useSelector(
+    (state) => state.quorum.extendedPublicKeyImporters
+  );
+
+  const keyOriginsWithName = Object.values(extendedPublicKeyImporters).map(
```

Comment:
maybe this should be a `useMemo` hook since `extendedPublicKeyImporters` can change when it's updated in redux.

Reply:
Fixed: 7d36214